### DISCUSSION TITLE
DevExpress.Xtrarport Error : DXXRDV Error

### DISCUSSION THREAD

**Discussion by Hptlive** [serenity-is/Serenity#6378] [Help]
Dear Sir, 
I am using the latest Startsharp 6.0. I have integrated the Devexpress Reports. 

The report has been added, and it is also displaying data from Model. The only issue is when we click on Print ,or Download in PDF,
it generate DXXRDV error. Can you please guide what settings we are missing.
We get following error.

POST https://localhost:58211/DXXRDV 400

Thanks

**Comment by wezmag** [comment]
Hi there,
Although I don't have any experience on Devexpress Reports, but I think the error has something to do with Antiforgery Token.
Try add `[IgnoreAntiforgeryToken]` on your action or controller.
